packages/shared/src/components/squads/Comment.tsx

```diff
@@ -63,6 +65,7 @@ export function SquadComment({
   const [commentary, setCommentary] = useState(form.commentary);
   const [link, setLink] = useState(preview.url);
   const [linkHint, setLinkHint] = useState(preview.url);
+  const isMobile = !useMedia([tablet.replace('@media ', '')], [true], false);
```

Comment:
Should we not use `const { sidebarRendered } = useSidebarRendered();`
It's a more condensed hook I think we preferred for mobile definition. (@sshanzel correct?)

Reply:
Same as here https://github.com/dailydotdev/apps/pull/1868#discussion_r1231976391